src/api/java/Context.java

```diff
@@ -322,13 +326,6 @@ public Constructor mkConstructor(Symbol name, Symbol recognizer,
 
     /**
      * Create a datatype constructor. 
-     * @param name  
-     * @param recognizer  
-     * @param fieldNames  
-     * @param sorts  
-     * @param sortRefs 
-     * 
-     * @return
```

Comment:
Instead of removing these, can you copy over the parameter descriptions from the previous function?